source/Calamari.Shared/Integration/Packages/Download/Scripts/DockerPull.ps1

```diff
@@ -1,14 +1,20 @@
+function IsDockerAvailable() {
+    $service = $(Get-Service -Name 'docker' -ErrorAction SilentlyContinue)
+    $serviceRunning = ($service -ne $null -and $service.Status -eq 'Running')
+    $process = $(Get-Process 'com.docker.proxy' -ErrorAction SilentlyContinue)
```

Comment:
This will fail if the Docker daemon is not running on the same machine. Remember, the `docker` binary can be configured to talk to a daemon anywhere, either via a pipe or TCP. It doesn't have to be local.